chapters/ch05/3.semantic-knowledge-graph.ipynb

```diff
@@ -13,18 +13,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys\n",
-    "\n",
-    "sys.path.append('../..')\n",
-    "sys.path.append(\"webserver\")\n",
-    "\n",
-    "from aips import get_engine, get_semantic_knowledge_graph as get_skg\n",
     "import json\n",
+    "\n",
     "from pyspark.sql import SparkSession\n",
     "\n",
+    "from aips import get_engine\n",
+    "from aips import get_semantic_knowledge_graph as get_skg\n",
+    "\n",
     "spark = SparkSession.builder.appName(\"AIPS\").getOrCreate()\n",
     "\n",
-    "engine = get_engine()"
+    "engine = get_engine(\"solr\")"
    ]
```

Comment:
If it's not hard coded here, than in listing 5.10 we'd potentially query the wrong engine. But yes, otherwise it would work because `get_skg` always returns a solr skg

Reply:
Dual indexing for ch5 has removed the need for hardcoding